homeassistant/components/reolink/switch.py

```diff
@@ -216,6 +216,15 @@ class ReolinkChimeSwitchEntityDescription(
         value=lambda api, ch: api.baichuan.privacy_mode(ch),
         method=lambda api, ch, value: api.baichuan.set_privacy_mode(ch, value),
     ),
+    ReolinkSwitchEntityDescription(
+        key="hardwired_chime_enabled",
+        cmd_key="483",
+        translation_key="hardwired_chime_enabled",
+        entity_category=EntityCategory.CONFIG,
```

Comment:
I would add the disabled by default = True.
This because it can make the chime rattle 1x during statup of HomeAssistant.